sequencer/api/migrations/sqlite/V602__redefine_stake_table_events.sql

```diff
@@ -0,0 +1,17 @@
+DROP TABLE stake_table_events;
+
+-- Stores the stake table events from the contract.  
+-- Each event is uniquely identified by a combination of `l1_block` and `log_index`
+CREATE TABLE stake_table_events (
+  l1_block BIGINT NOT NULL,
+  log_index BIGINT NOT NULL,
+  event JSONB NOT NULL,
+  PRIMARY KEY (l1_block, log_index)
+);
+
+-- Tracks the last L1 block that has been finalized and whose events have been processed and stored.  
+-- This tracking is necessary to determine the starting point for fetching new contract events. 
+CREATE TABLE stake_table_events_l1_block (
+  id INTEGER PRIMARY KEY CHECK (id = 0),
```

Comment:
Is there a reason we use several different ways of doing singleton tables, e.g.

https://github.com/EspressoSystems/espresso-network/blob/ee934e1cba4dc7b84b770a74c8ebf79fced2a0fd/sequencer/api/migrations/sqlite/V203__next_epoch_qc.sql#L2

Reply:
There is no reason.  I can change it to the other way with id as bool, but I don't think it matters

Reply:
It's just a bit easier to read if we're consistent.